wrangler-transform/src/main/java/co/cask/wrangler/Wrangler.java

```diff
@@ -342,10 +342,9 @@ public void initialize(TransformContext context) throws Exception {
         ConfigDirectiveContext dContext = new ConfigDirectiveContext(url);
         recipe.initialize(dContext);
       } else {
-        LOG.warn(
-          String.format("Stage:%s - Context is set to default, no aliasing and restriction would be applied.",
-                        getContext().getStageName())
-          );
+        // this is normal in cloud environments
```

Comment:
If it is normal then why do we need to log it?

Reply:
because it's not normal if you run with the native profile. I'm not actually sure what aliasing and restriction are, but I would imagine it's good to know if you're using those features.

Reply:
I see.